analytics/tests/test_counts.py

```diff
@@ -179,7 +179,7 @@ def create_direct_message_group_with_recipient(
         self, **kwargs: Any
     ) -> tuple[DirectMessageGroup, Recipient]:
         self.name_counter += 1
-        defaults = {"huddle_hash": f"hash{self.name_counter}"}
+        defaults = {"huddle_hash": f"hash{self.name_counter}", "group_size": 4}
```

Comment:
I added `group_size` as 4 here just to not let it be None while creating the object for the tests (I don't find any Subscriptions objects being created). It is not being tested anywhere anyways.

Reply:
Yeah that's probably OK.